src/ai/backend/client/cli/session.py

```diff
@@ -995,23 +993,22 @@ def print_state(session_name_or_id: str, current_state_idx: int):
             print_state(session_name_or_id, current_state_idx=-1)
```

Comment:
Could you explain what is the purpose of using the state index?

Reply:
Since the state changes in serial order, we can use index to check if one kernel/session has reached to specific state. In this line, `current_state_idx=-1` means that the session is not reached to any state yet; even to the initial state.
Would it be better to set the default value of `current_state_idx` to `-1` and make it implicit?